coalib/processes/Processing.py

```diff
@@ -335,10 +335,10 @@ def instantiate_processes(section,
                              the same for each object.
     """
     filename_list = collect_files(
-        path_list(section.get('files', "")),
+        glob_list(section.get('files', "")),
         log_printer,
-        ignored_file_paths=path_list(section.get('ignore', "")),
-        limit_file_paths=path_list(section.get('limit_files', "")))
+        ignored_file_paths=glob_list(section.get('ignore', "")),
+        limit_file_paths=glob_list(section.get('limit_files', "")))
```

Comment:
No, `section.get('files', "")` for example would return stuff such as `coalib/**/*.py` and so on. When this is passed to `glob_list`, it escapes the origin alone (so the path to the `coafile`) and then appends `coalib/**/*.py` without any change. Then it starts globbing and getting matches, which would work proper.